scripts/questions_gen.py

```diff
@@ -245,6 +245,9 @@ async def questions_eval(
         res_row["passing"] = result.passing
         res_row["feedback"] = result.feedback
         res_row["score"] = result.score
+
```

Comment:
not rebased perhaps? it is from different PR

Reply:
Yeah had some issues, ignore that